tests/0080-admin_ut.c

```diff
@@ -1756,10 +1807,12 @@ static void do_test_ListConsumerGroupOffsets(const char *what,
         rd_kafka_queue_t *q;
 #define MY_LIST_CGRPOFFS_CNT 1
         rd_kafka_AdminOptions_t *options = NULL;
-        rd_kafka_topic_partition_list_t *empty_cgoffsets_list;
+        rd_kafka_topic_partition_list_t *cgoffsets_empty_list;
```

Comment:
I think its better to have more verbose name. Something like `listcgoffsets_empty_partitions`. The current name suggests that we are dealing with offsets here but we are just concerned about partitions.

Reply:
It's because it's the list used by cgoffsets_empty, uniformed to the name like in do_test_AlterConsumerGroupOffsets.

Reply:
I feel it still gives an impression that we are dealing with offsets but here we want verify that we don't have duplicate partitions. In alter API, it made sense as we don't want duplicate offset.

Reply:
Here too, we were checking that in J. is returns a Map with TopicPartition so it's better that the partitions in the request are unique too to avoid having a different count

Reply:
I am just talking about the variable name. `cgoffsets_duplicate` implies that we are talking about offsets. In ListConsumerGroupOffsets, we don't want to have duplicate partitions, so I feel its better to change it to something like `*partitions_duplicate*`. Nothing critical though.